pkgs/applications/misc/obinskit/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchurl 
+, xlibs
+, libxkbcommon
+, libudev
+, gcc-unwrapped
+, electron_3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obinskit";
+  version = "1.1.1";
+
+  src = fetchurl {
+    url = "http://releases.obins.net/occ/linux/tar/ObinsKit_${version}_x64.tar.gz";
+    sha256 = "0052m4msslc4k9g3i5vl933cz5q2n1affxhnm433w4apajr8h28h";
+  };
+
+  unpackPhase = ''
+    tar -xzf $src
+    export src='ObinsKit_1.1.1_x64'
+  '';
+
+  installPhase = ''
+    mkdir -p $out/opt/obinskit
+    install $src/icudtl.dat $out/opt/obinskit/
+    install $src/natives_blob.bin $out/opt/obinskit/
+    install $src/v8_context_snapshot.bin $out/opt/obinskit/
+    install $src/blink_image_resources_200_percent.pak $out/opt/obinskit/
+    install $src/content_resources_200_percent.pak $out/opt/obinskit/
+    install $src/content_shell.pak $out/opt/obinskit/
+    install $src/ui_resources_200_percent.pak $out/opt/obinskit/
+    install $src/views_resources_200_percent.pak $out/opt/obinskit/
+    cp -r $src/resources $out/opt/obinskit/
+    cp -r $src/locales $out/opt/obinskit/
+
+    mkdir -p $out/bin
+
+    cat <<-EOF > $out/bin/obinskit
+    #!/usr/bin/env bash
+
+    export LD_LIBRARY_PATH="${libxkbcommon}/lib:${xlibs.libXt}/lib:${libudev.lib}/lib:${gcc-unwrapped.lib}/lib"
+
+    exec ${electron_3}/bin/electron $out/opt/obinskit/resources/app.asar
+    EOF
+
+    chmod +x $out/bin/obinskit
```

Comment:
Using `makeWrapper` will eliminate the need for this entirely.

```
nativeBuildInputs = [
  makeWrapper
];

...

makeWrapper ${electron_3}/bin/electron $out/bin/obinskit \
  --add-flags $out/opt/obinskit/resources/app.asar \
  --prefix LD_LIBRARY_PATH : "${libxkbcommon}/lib:${xlibs.libXt}/lib:${libudev.lib}/lib:${gcc-unwrapped.lib}/lib"
```

also think `gcc-unwrapped` should be `stdenv.cc.cc`. Also `xlibs` is an alias to a new attribute `xorg` so you should use `xorg.libXt`. CI evaluation fails on this pr because of the use of the alias.